mysite/_config/blog.yml

```diff
@@ -1,63 +0,0 @@
----
-Name: mywidgetsconfig
-Only:
-  moduleexists: widgets
----
-# Disable if you do not use widgets on your site
-SiteTree:
-  extensions:
-    - WidgetPageExtension
-
----
-Name: mycommentsextension
-Only:
-  moduleexists: comments
----
-# Enable page comments on the site by default, including frontend moderation / approval
-SiteTree:
-  extensions:
-    - CommentsExtension
-  comments:
-    enabled: false
-    frontend_moderation: true
-    require_moderation_nonmembers: true
-    require_moderation_cms: true
-    require_login: false
-    require_login_cms: true
-    nested_comments: true
-    order_comments_by: '"Created" ASC'
-
----
-Name: myblogconfig
-Only:
-  moduleexists: blog
----
-# Customise the email notification template here
-BlogPost:
-  default_notification_template: 'BlogCommentEmail'
-  comments:
-    enabled: true
-
----
-Name: akismetconfig
-Only:
-  moduleexists: akismet
----
-# Customise your akismet configuration here
-SiteConfig:
-  extensions:
-    - AkismetConfig
-# Allows spam posts to be saved for review if necessary
-AkismetSpamProtector:
-  save_spam: true
-
----
-Name: mycommentspamprotection
-Only:
-  moduleexists: spamprotection
-  classexists: CommentingController
----
-# Enable spam protection for comments by default
-CommentingController:
-  extensions:
-    - CommentSpamProtection
```

Comment:
Do we have a plan for where this config is going if we're removing it from here?

Reply:
Whichever recipe `silverstripe/comments` ends up in, I imagine into `cwp-recipe-blog`

Reply:
Worth noting that the blog recipe is going to become a SilverStripe recipe, not CWP specific

Reply:
Ah right. I guess there's no harm in random extra configuration. Can always add a comment about it :>

Reply:
Having reviewed the contents of the configuration, I can't see any CWP specific - even the email template is generic enough to be included in any generic blog recipe, which can in turn then be a composite addition to this one.

Reply:
I don't think we should delete anything until we have somewhere else to put it. Perhaps we can just leave this stuff for now and focus on the test suites?

Reply:
As you know (but I want to address here anyway for public consumption), this is now is `silverstripe/recipe-blog` as I thought effectively 'double handling' this was unnecessary.